server/auth.go

```diff
@@ -31,18 +31,18 @@ func (s *Server) registerAuthRoutes(g *echo.Group) {
 		}
 		user, err := s.Store.FindUser(ctx, userFind)
 		if err != nil && common.ErrorCode(err) != common.NotFound {
-			return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to find user by username %s", signin.Username)).SetInternal(err)
+			return echo.NewHTTPError(http.StatusInternalServerError, "Your login credentials are incorrect please check")
```

Comment:
How about `Incorrect login credentials, please try again`

Reply:
@boojack changed to  `Incorrect login credentials, please try again`  u can check